six/seven.py

```diff
@@ -0,0 +1,26 @@
+DIGIT_LETTERS = {
+    '2': ['A', 'B', 'C'],
+    '3': ['D', 'E', 'F'],
+    '4': ['G', 'H', 'I'],
+    '5': ['J', 'K', 'L'],
+    '6': ['M', 'N', 'O'],
+    '7': ['P', 'Q', 'R', 'S'],
+    '8': ['T', 'U', 'V'],
+    '9': ['W', 'X', 'Y', 'Z']
+}
+
+
+def get_mnemonics(digits):
+    if len(digits) > 1:
+        prev_sequences = get_mnemonics(digits[:-1])
+        digit = digits[-1]
+        letters = DIGIT_LETTERS[digit]
+        
+        new_sequences = []
+        for seq in prev_sequences:
+            for letter in letters:
+                new_sequences.append(seq + letter)
+    else:
```

Comment:
This is a style thing, but I would have made the first two lines

```python
if len(digits) ==1:
  return DIGIT_LETTERS[digits]
```
and then unindent the rest of the function

Reply:
fair enough